backend/db/migrations/20201029063356-addProducts.js

```diff
@@ -0,0 +1,29 @@
+const tableName = 'products'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    const isPostgres = queryInterface.sequelize.options.dialect === 'postgres'
+
+    return queryInterface.sequelize.transaction(async () => {
+      await queryInterface.createTable(tableName, {
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true
+        },
+        product_id: Sequelize.STRING,
+        stock_left: Sequelize.INTEGER,
```

Comment:
Should this be an unsigned integer since it should never go negative?

Reply:
Yep, will change it

Reply:
There seems to no unsigned int in postgres. But have added a less-than-zero check before we update the count. That should be fine for now I think